dotcom-rendering/src/web/components/Liveness.importable.tsx

```diff
@@ -179,6 +179,42 @@ export const Liveness = ({
 		observer.observe(topOfBlog);
 	}
 
+	/**
+	 * This useEffect sets up a listener for when the page is backgrounded or restored. We
+	 * do this so that any new blocks that were fetched while the blog was in the
+	 * background are animated in at the point when focus is restored
+	 */
+	useEffect(() => {
+		const handleVisibilityChange = () => {
+			// The blog was either hiiden or has become visible
+			if (
+				// If we're returning to a blog that has pending blocks and the reader
+				// is at the top of the page then...
+				document.visibilityState === 'visible' &&
+				numHiddenBlocks > 0 &&
+				topOfBlogVisible()
+			) {
+				revealNewBlocks();
+				setNumHiddenBlocks(0);
+				setShowToast(false);
+			}
+		};
+
+		window.addEventListener(
+			'visibilitychange',
+			handleVisibilityChange,
+			false,
```

Comment:
Isn’t this the default value, anyways? If being explicit is what we’re after, shouldn’t we use the `options` object instead?

Reply:
I think you're right, it's not required so the simplest thing is to just not pass it. Done